test/harness/index.js

```diff
@@ -16,7 +16,14 @@
 
 'use strict';
 
-// WPT's assert_throw uses a list of predefined, hardcoded know errors. Since
+let testNum = (function() {
```

Comment:
It doesn't really provide a stable test identifier for **all** the test cases, but it does for all the ones that are automatically generated from the wast core tests. js-api and HTML tests still manage their id by themselves.

This is needed by the WPT harness. The [wptrunner](https://github.com/w3c/wptrunner) has the ability to automatically generate expectation files, which encode whether a single test is known to be passing or failing when run in a specific browser. For this to work, the (filename, test_id) couple needs to be unique, which makes sense. This is the reason why I've added the unique test id.